controllers/release/release_adapter_test.go

```diff
@@ -43,10 +44,11 @@ var _ = Describe("Release Adapter", Ordered, func() {
 	var (
 		adapter *Adapter
 
-		release             *appstudiov1alpha1.Release
-		releaseStrategy     *appstudiov1alpha1.ReleaseStrategy
-		releaseLink         *appstudiov1alpha1.ReleaseLink
-		applicationSnapshot *appstudioshared.ApplicationSnapshot
+		release              *appstudiov1alpha1.Release
+		releaseStrategy      *appstudiov1alpha1.ReleaseStrategy
+		releasePlan          *appstudiov1alpha1.ReleasePlan
+		releaseAdmissionPlan *appstudiov1alpha1.ReleaseAdmissionPlan
+		applicationSnapshot  *appstudioshared.ApplicationSnapshot
```

Comment:
no longer anal about alphabetical order huh?

Reply:
Not an order I added but an order I fixed :wink: